docs/USING_ADVANCED.md

```diff
@@ -61,6 +61,14 @@ console.log(marked(markdownString));
 |walkTokens   |`function`  |`null`|v1.1.0|A function which is called for every token. See [extensibility](/using_pro) for more details.|
 |xhtml       |`boolean` |`false`  |v0.3.2   |If true, emit self-closing HTML tags for void elements (&lt;br/&gt;, &lt;img/&gt;, etc.) with a "/" as required by XHTML.|
 
+<h2 id="inline">Inline Markdown</h2>
+
+You can parse inline markdown by running markdown through `marked.parseInline`.
+
+```js
+marked.parseInline(markdownString, options)
+```
```

Comment:
Can you use the same example from your PR description here in the docs? That might help explain the purpose.